modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/construction/ConstructionService.scala

```diff
@@ -39,5 +41,12 @@ object ConstructionService {
       .map(TransactionIdentifier(_))
       .attemptT
       .leftMap(_ => MalformedTransaction)
+
+    def getAccountIdentifiers(operations: List[Operation]): Option[NonEmptyList[AccountIdentifier]] = {
+      val accountIdentifiers =
+        operations.filter(_.amount.exists(_.value.value.value < 0)).flatMap(_.account)
```

Comment:
`_.value.value.value` looks weird and I'd think about supportive functions in `AmountValue` like `def isNegative`/`def isPositive`. Then, this would look like `_.amount.exists(_.value.isNegative)`

Reply:
Added `isNegative` to `AmountValue`.